index.js

```diff
@@ -75,26 +61,26 @@ const renderSubMenu = (options, param2, index) => {
 
```

Comment:
lines 59 and 60 should be in global scope

Reply:
Solved